sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -181,6 +181,7 @@ enum class image_type : /* unspecified */ {
   mipmap,
   array,
   cubemap,
+  gather,
```

Comment:
We currently don't have a good, central description of all of these types, we'll want to add that in a future patch.